include/klee/Internal/Module/KInstruction.h

```diff
@@ -44,7 +44,8 @@ namespace klee {
 
   public:
     virtual ~KInstruction();
-    void printFileLine(llvm::raw_ostream &);
+    void printFileLine(llvm::raw_ostream &) const;
+    std::string printFileLine() const;
```

Comment:
The `printFileLine(llvm::raw_ostream &)` variant should be removed.

`printFileLine()` is the result of a refactor in 6f20f660 . Really though given the way it is currently being used there should only be one method `std::string getFileLine() const;` (or whatever we want to call it). 

The single use is 

```c++
  if (!optionIsSet(DebugPrintInstructions, STDERR_COMPACT) &&
      !optionIsSet(DebugPrintInstructions, FILE_COMPACT)) {
    (*stream) << "     ";
    state.pc->printFileLine(*stream);
    (*stream) << ":";
  }
```

that can just be

```c++
  if (!optionIsSet(DebugPrintInstructions, STDERR_COMPACT) &&
      !optionIsSet(DebugPrintInstructions, FILE_COMPACT)) {
    (*stream) << "     " << state.pc->getFileLine() << ":";
  }
```

With `printFileLine(llvm::raw_ostream &)` removed there is no longer an argument for using `std::string printFileLine() const;` so we can give it a more sensible name like `std::string getSourceLocation() const;`